crates/store/re_types/src/components/media_type_ext.rs

```diff
@@ -260,5 +297,79 @@ fn test_media_type_extension() {
     assert_eq!(MediaType::markdown().file_extension(), Some("md"));
     assert_eq!(MediaType::plain_text().file_extension(), Some("txt"));
     assert_eq!(MediaType::png().file_extension(), Some("png"));
+    assert_eq!(MediaType::rvl().file_extension(), Some("rvl"));
     assert_eq!(MediaType::stl().file_extension(), Some("stl"));
 }
+
+#[test]
+fn test_guess_from_data() {
```

Comment:
most of this test is for `MediaType::rvl()`, which I think is fine. No need to test the GLB/stl here in my opinion. Test can also be renamed to `test_guess_from_data_rvl`